src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -14,6 +14,9 @@ internal static class CompletionOptions
         public static readonly PerLanguageOption<bool> TriggerOnTyping = new PerLanguageOption<bool>(FeatureName, "TriggerOnTyping", defaultValue: true);
         public static readonly PerLanguageOption<bool> TriggerOnTypingLetters = new PerLanguageOption<bool>(FeatureName, "TriggerOnTypingLetters", defaultValue: true);
 
+        public static readonly PerLanguageOption<int> AddNewLineOnEnterAfterFullyTypedWord =
```

Comment:
Hrmm... i'm not certain what i think is best.  Part of me things we should ahve this new otion, which serializes under the name "EnterKeyBehavior".  We then keep the old option around and migrate it forward if we see that it is there.    We put an Obsolete attribute on the old one to tell people to not use it anymore though.